File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Commons/ConservationModel.ts
import ConservationStatusType = require("./ConservationStatusType");
import ConservationType = require("./ConservationType");

interface ConservationModel {
    UniqueId: number;
    EntityType: string;
    Message: string;
    SendDate?: Date;
    Uri: string;
    Status: ConservationStatusType;
    Type: ConservationType;
}

export = ConservationModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Conservations/ConservationModel.cs
using System;
using VecompSoftware.DocSuiteWeb.Model.Conservations;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Conservations
{
    public class ConservationModel
    {
        #region [ Constructor ]
        #endregion

        #region [ Properties ]
        public string EntityType { get; set; }
        public ConservationStatus Status { get; set; }
        public string Message { get; set; }
        public ConservationType Type { get; set; }
        public DateTimeOffset? SendDate { get; set; }
        public string Uri { get; set; }
        public Guid UniqueId { get; set; }
        public DateTimeOffset RegistrationDate { get; set; }
        public string RegistrationUser { get; set; }
        public DateTimeOffset? LastChangedDate { get; set; }
        public string LastChangedUser { get; set; }
        #endregion
    }
}
